### PR TITLE
Cover badger tree deficit and future badger graviaura emissions

### PR DESCRIPTION
**Pull request by petrovska-petro**
Tackles #620, **uniquely for now the badger deficit**

```
brownie run issue/620/resolve_badger_deficit
```

**Context**: [top-10-claimable](https://gist.github.com/petrovska-petro/9d04f986a2827dea3979171ed63bf40b)

### CODE REVIEW

File: scripts/issue/620/resolve_badger_deficit.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+def main():
+    vault = GreatApeSafe(registry.eth.badger_wallets.treasury_vault_multisig)
+    badger = interface.ERC20(registry.eth.treasury_tokens.BADGER, owner=vault.account)
+
+    vault.take_snapshot([badger])
+
+    # current deficit is ~198k badger + 19k badger which will be emitted between week 27 - 31
+    # https://github.com/Badger-Finance/badger-multisig/issues/620#issuecomment-1177909957
+    badger.transfer(registry.eth.drippers.tree_2022_q3, 223_000e18)
```
Comment (gosuto-inzasheru) [inline scripts/issue/620/resolve_badger_deficit.py:15]: i think you failed to take the pending badger from the dripper into account: https://etherscan.io/tx/0xe4c58295b26e807fd9d75091a43354338303229e98403701d55083acacc36518
```suggestion
    # current deficit is ~198k badger + 19k badger which will be emitted between week 27 - 31
    # https://github.com/Badger-Finance/badger-multisig/issues/620#issuecomment-1177909957
    badger.transfer(registry.eth.drippers.tree_2022_q3, 190_000e18)
```

Comment (petrovska-petro) [reply scripts/issue/620/resolve_badger_deficit.py]: great point. i consider that amount now via latest commit: [8091b30](https://github.com/Badger-Finance/badger-multisig/pull/621/commits/8091b3093ba3e6b707c7068912e1e813a2f28342)

**Review by gosuto-inzasheru — APPROVED** [review]
looks good, one accounting problem less to worry about!

**Review by jalbrekt85 — APPROVED** [review]
deficit calc lgtm

**Pull request merged**